assets/in

```diff
@@ -66,14 +93,17 @@ if test "$depth" -gt 0 2> /dev/null; then
   depthflag="--depth $depth"
 fi
 
-branch="pull-requests/${prq_id}/merge"
-
 log "Cloning $uri in $destination"
 git clone $depthflag "$uri" "$destination"
 cd "$destination"
 
-git fetch $depthflag origin "+refs/${branch}:refs/remotes/origin/${branch}"
-git checkout -B $branch origin/$branch
+# TODO cleanup
+#branch="pull-requests/${prq_id}/merge"
+#git fetch $depthflag origin "+refs/${branch}:refs/remotes/origin/${branch}"
+#git checkout -B $branch origin/$branch
+
+git fetch $depthflag origin "${prq_from_branch}"
+git checkout -B "${prq_from_branch}" "origin/${prq_from_branch}"
```

Comment:
This changes the behavior of the resource.

The original code would checkout the special commit refs/pull-requests/$id/merge which is an automatic merge of the branches involved in the pull request.

To maintain the same behavior, we should checkout the to_branch and immediately merge the from_branch.

Reply:
Have to say I am not a git pro. In my understanding you want to do the following:

Pseudo:
```
git fetch […] from_branch
git fetch […] to_branch
git checktout -B to_branch
git merge from_branch

?? commit // What to do on merge conflict?
```

Can you please confirm my assumption or state the git calls to be integrated?